airbyte_cdk/sources/streams/concurrent/adapters.py

```diff
@@ -302,6 +302,12 @@ def read(self) -> Iterable[Record]:
                         stream_name=self.stream_name(),
                         associated_slice=self._slice,  # type: ignore [arg-type]
                     )
+                elif isinstance(record_data, AirbyteMessage) and record_data.record is not None:
```

Comment:
I'm not sure how this works with the file based universal file transfer records. It seems like right now, we are excluding those from this condition. Is that expected?

Reply:
file based connectors have their own [facade](https://github.com/airbytehq/airbyte-python-cdk/blob/main/airbyte_cdk/sources/file_based/stream/concurrent/adapters.py). I'll add a comment mentioning we need to support them here if we want to reconcile the facades.